h/schemas/annotation.py

```diff
@@ -305,10 +305,16 @@ def _target_selectors(targets):
     """
     # Any targets other than the first in the list are discarded.
     # Any fields of the target other than 'selector' are discarded.
+    selectors = []
     if targets and "selector" in targets[0]:
-        return targets[0]["selector"]
+        selectors = targets[0]["selector"]
 
-    return []
+    for target_selector in selectors:
+        if suffix := target_selector.get("suffix"):
```

Comment:
This could happen on prefix I suppose, I'd prefer to mark the sentry exception as solved and add this check to other fields.